src/View/SSViewer.php

```diff
@@ -253,9 +253,9 @@ public static function get_templates_by_class($classOrObject, $suffix = '', $bas
             $templates[] = $template;
             $templates[] = ['type' => 'Includes', $template];
 
-            // If the class is "Page_Controller", look for Page.ss
-            if (stripos($class, '_controller') !== false) {
-                $templates[] = str_ireplace('_controller', '', $class) . $suffix;
+            // If the class is "PageController" (PSR-2 compatibility) or "Page_Controller" (legacy), look for Page.ss
+            if (preg_match('/^(?<name>.+[^\\])_?Controller$/i', $class, $matches)) {
```

Comment:
Sorry, you'll need to quad-escape this `[^\\\\]`. Which collapses to `[^\\]` when evaluated by PHP, and `[^\]` when evaluated by regexp :D